packages/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleValue.tsx

```diff
@@ -42,49 +43,37 @@ function GridFilterInputMultipleValue(props: GridFilterInputMultipleValueProps)
     [applyValue, item, type],
   );
 
+  const handleInputChange = React.useCallback(
+    (event: React.SyntheticEvent, value: string) => {
+      if (value === '') {
+        setOptions([]);
+      } else {
+        setOptions([value]);
+      }
+    },
+    [setOptions],
+  );
+
+  const BaseAutocomplete = rootProps.slots.baseAutocomplete as React.JSXElementConstructor<
+    AutocompleteProps<string, true, false, true>
+  >;
```

Comment:
We can't have type parameters directly on `GridBaseSlots['baseAutocomplete']`, the parameters would need to be on `GridBaseSlots` and that would make using that type extremely tedious.